src/crimson/osd/object_context_loader.h

```diff
@@ -223,8 +233,16 @@ class ObjectContextLoader {
   template<RWState::State State>
   load_obc_iertr::future<> with_clone_obc_only(ObjectContextRef head,
                                                hobject_t clone_oid,
-                                               with_obc_func_t&& func,
-                                               bool resolve_clone = true);
+                                               with_obc_func_t func,
+                                               bool resolve_clone = true) {
+    LOG_PREFIX(ObjectContextLoader::with_clone_obc_only);
+    SUBDEBUGDPP(osd, "{}", dpp, clone_oid);
+    auto manager = get_obc_manager(clone_oid, resolve_clone);
+    manager.head_state.obc = head;
+    manager.head_state.obc->append_to(obc_set_accessing);
```

Comment:
Alternatively, can we drop with_clone_obc_only? It was introduced to avoid "inner" with_lock calls. I think we should be able to always use with_obc now.

Reply:
It's used in PGBackend::rollback.  I'd like to avoid refactoring that part as part of this PR, we can do further cleanups later.